market/connector/docusign/demo/eSignatureDemo/pom.xml

```diff
@@ -30,6 +30,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
```

Comment:
For what is this lib?

Reply:
it provide many encrypting algorithms which are not by default part of all JREs. But more than that, it was the only reproducible way to read the private-key PEM files created by docusign. 
In theory the algorithm need is RSA-256 and should work without this library. But I could not get it to work without it ... maybe there is a way of converting the PEM to another format or the like which I do not know .

Reply:
ok 👍